src/test/java/org/example/JewelsAndStonesTest.java

```diff
@@ -0,0 +1,12 @@
+package org.example;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class JewelsAndStonesTest {
+    @Test
+    public void testCountJewelsInStones(){
+        Assert.assertEquals(3, JewelsAndStones.countJewelsInStones("aA", "aAAbbbb"));
+        Assert.assertEquals(0, JewelsAndStones.countJewelsInStones("z", "ZZ"));
+    }
```

Comment:
this is a good approach for testing positives, negatives, and some edge cases. for example cauntJewels_JewelsEmpty_BadRequestIsThrown() {... }. Keep calling the Given-When-Then pattern. It would be clear what is being tested and what the expected result is